pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -40,6 +43,8 @@ const (
 	SelfProvisionerRoleName   = "self-provisioner"
 	BasicUserRoleName         = "basic-user"
 	StatusCheckerRoleName     = "cluster-status"
+	ImagePullerRoleName       = "image-puller"
```

Comment:
I don't see why we'd ever want to let a user manipulate these roles, but I can imagine cases where introduce new resources/subresources and need to grant access to builders.